packages/mobile/src/components/challenge-rewards-drawer/ChallengeRewardsDrawerContent.tsx

```diff
@@ -102,98 +135,159 @@ export const ChallengeRewardsDrawerContent = ({
+    const formattedLabel = isClose ? (
+      label
+    ) : (
+      <Text>
+        {label}&nbsp;
+        <Text color='subdued'>{claimableDate.format('(M/D)')}</Text>
+      </Text>
+    )
+    return {
+      ...item,
+      label: formattedLabel
+    }
+  }, [])
 
-  return (
-    <View style={styles.content}>
-      {isVerifiedChallenge ? (
-        <ChallengeDescription
-          task={messages.taskVerified}
-          taskIcon={
-            <IconVerified
-              style={styles.subheaderIcon}
-              fill={palette.staticPrimary}
-              fillSecondary={palette.staticWhite}
-            />
-          }
-          description={description}
+  const renderCooldownSummaryTable = () => {
+    if (isCooldownChallenge && !isCooldownChallengesEmpty) {
+      return (
+        <SummaryTable
+          title={messages.upcomingRewards}
+          items={formatCooldownChallenges(cooldownChallenges).map(formatLabel)}
+          summaryItem={summary}
+          secondaryTitle={messages.audio}
+          summaryLabelColor='accent'
+          summaryValueColor='default'
         />
-      ) : (
-        <ChallengeDescription description={description} />
-      )}
-      <View style={styles.statusGrid}>
-        <View style={styles.statusGridColumns}>
-          <ChallengeReward amount={amount} subtext={messages.audio} />
-          {showProgressBar ? (
-            <View style={styles.progressCell}>
-              <Text
-                color='neutralLight4'
-                style={[styles.subheader, styles.progressSubheader]}
-                weight='heavy'
-                textTransform='uppercase'
-              >
-                {messages.progress}
-              </Text>
-              <ProgressBar progress={currentStep} max={stepCount} />
-            </View>
-          ) : null}
-        </View>
-        <View
-          style={[
-            styles.statusCell,
-            hasCompleted ? styles.statusCellComplete : null
-          ]}
-        >
-          <Text
+      )
+    }
+    return null
+  }
+
+  return (
+    <>
+      <ScrollView style={styles.content}>
+        {isVerifiedChallenge ? (
+          <ChallengeDescription
+            task={messages.taskVerified}
+            taskIcon={
+              <IconVerified
+                style={styles.subheaderIcon}
+                fill={palette.staticPrimary}
+                fillSecondary={palette.staticWhite}
+              />
+            }
+            description={description}
+          />
+        ) : (
+          <ChallengeDescription description={description} />
+        )}
+        <View style={styles.statusGrid}>
+          <View style={styles.statusGridColumns}>
+            <ChallengeReward amount={amount} subtext={messages.audio} />
+            {showProgressBar ? (
+              <View style={styles.progressCell}>
+                <Text
+                  color='subdued'
+                  style={[styles.subheader, styles.progressSubheader]}
```

Comment:
the custom styling is for layout in particular the margin bottom which i don't think text can do alone. i think you would need to make broader changes around the container? lmk otherwise